src/main/java/moriyashiine/aylyth/common/data/tag/ModPotionTags.java

```diff
@@ -1,4 +1,4 @@
-package moriyashiine.aylyth.common.registry.tag;
+package moriyashiine.aylyth.common.data.tag;
```

Comment:
Rename `ModPotionTags` to be in-line with new tag class prefixes, ie `AylythPotionTags`